accounts/views.py

```diff
@@ -935,38 +946,45 @@ def old_user_link_redirect(request):
 @login_required
 def email_reset(request):
     if request.method == "POST":
-        form = EmailResetForm(request.POST, user=request.user)
```

Comment:
This changes depend on whether `EmailResetForm` keeps the `clean_password` method (see my comment above).